src/components/inventoryList/inventoryList.js

```diff
@@ -53,10 +60,21 @@ class InventoryList extends React.Component {
         cellData
       });
 
+      const hasGuests = cellData?.numberOfGuests > 0;
+      const guestsId = cellData?.subscriptionManagerId;
       updatedColumnHeaders = columnHeaders;
 
       return {
-        cells
+        cells,
+        expandedContent: hasGuests && guestsId && (
+          <GuestsList
+            key={guestsId}
+            filterGuestsData={filterGuestsData}
+            numberOfEntries={cellData?.numberOfGuests}
```

Comment:
Optional:
Consider aligning the attribute and object property names to both "numberOfGuests"